insect.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Name=Insect
+GenericName=Scientific calculator
+TryExec=insect
+Exec=insect
+Icon=insect
```

Comment:
Do .desktop files work with PNG icons?

Reply:
How about shipping this icon without "insect" written on it? https://insect.sh/media/insect-favicon.svg
The one with "insect" written on it would seem become a small icon in an app launcher, since the name of app will take some space.

Reply:
Fine by me. @sharkdp do you have any issues with shipping `insect-favicon.svg` and removing `insect.png` from the NPM package? If you don't, I would also prefer to rename `insect-favicon.svg` to something like `insect.svg`. Thoughts about that too?

Reply:
No, sounds like a good idea. Feel free to use that file from insect.sh.